app/src/main/res/layout/activity_baker_settings.xml

```diff
@@ -63,22 +63,34 @@
                         app:layout_constraintStart_toStartOf="parent"
                         app:layout_constraintTop_toBottomOf="@id/title" />
 
-                    <EditText
+                    <com.concordium.wallet.uicore.view.AmountEditText
                         android:id="@+id/transactionFeeValue"
                         style="@style/TextView.SingleLine"
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
                         android:layout_marginHorizontal="@dimen/activity_horizontal_margin_double"
                         android:layout_marginTop="@dimen/activity_horizontal_margin_half"
-                        android:background="@color/background_transparent"
-                        android:inputType="numberDecimal"
+                        android:background="@null"
                         android:imeOptions="actionDone"
+                        android:inputType="numberDecimal"
                         android:maxLength="15"
                         android:textAlignment="center"
+                        android:textSize="18sp"
                         app:layout_constraintEnd_toEndOf="parent"
                         app:layout_constraintStart_toStartOf="parent"
                         app:layout_constraintTop_toBottomOf="@id/transactionFeeTitle"
-                        tools:text="10%" />
+                        app:layout_constraintWidth="wrap_content"
```

Comment:
```suggestion
                        app:layout_constraintWidth="wrap_content"
                        app:layout_constraintWidth_min="10dp"
```
Can we add a min size for these Edit Views, so that if the amount is deleted, one can still click them?

Reply:
Makes sense, I think I'll also make clicking the "%" focus the corresponding field.

Reply:
I'll add the suggestion in the release branch not to wait for another review.